lib/3llo/entities.rb

```diff
@@ -11,7 +11,7 @@ module Entities
     Board = Struct.new(:id, :shortcut, :name)
     List = Struct.new(:id, :shortcut, :name)
     Card = Struct.new(:id, :shortcut, :name, :description, :short_url, :labels, :members, :list, keyword_init: true)
-    Label = Struct.new(:name, :color)
+    Label = Struct.new(:id, :shortcut, :name, :color)
```

Comment:
```suggestion
    Label = Struct.new(:id, :shortcut, :name, :color, keyword_init: true)
```

I think let's use keyword init.

Reply:
Added, please check it again.